README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-145-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-146-orange.svg?style=flat-square)](#contributors-)
```

Comment:
At the same time can you get back to: `<a href="#contributors-"><img src="https://img.shields.io/badge/all_contributors-146-orange.svg?style=flat-square" alt="All Contributors"></a>`

Reply:
Thanks for the hint @luigifab. I just ran the command in the PR template (`yarn all-contributors add @YOUR_NAME <types>`), which unfortunately only incremented the number. The link should be restored now. 👍